### PR TITLE
Remove unnecessary package pciutils from dependencies and purge on update

### PR DESCRIPTION
**Pull request by MichaIng**
Fix https://github.com/Fourdee/DietPi/issues/1068

I hope I implemented the purge on update the correct way? :)

### CODE REVIEW

**Comment by MichaIng** [comment]
And how do I change a commit instead of adding a new one every time? Lost connection with mobile during editing ;).

**Comment by Fourdee** [comment]
@MichaIng 

Spot on, thanks for this 👍 

> And how do I change a commit instead of adding a new one every time? Lost connection with mobile during editing ;).

I believe you can only continue to send commits, to resolve edits. Either that, or revert a change, then send another commit.

I tend to "commit away" until its how I want it 😉

File: dietpi/patch_file [hunk 1]
```diff
@@ -2650,6 +2650,12 @@ _EOF_
 		fi
 		#-------------------------------------------------------------------------------
 
+        elif (( $VERSION_CURRENT == 154 )); then
```
Comment (Fourdee) [inline dietpi/patch_file]: This wont have any effect until v155 is released. As the current version is v153 during the v154 update.

@MichaIng 

If you could remove this section and place the patch code in ```elif (( $VERSION_CURRENT == 153 )); then```, i'd be grateful and merge this commit.

**Comment by MichaIng** [comment]
Done.

As I didn't find a way to change/remove commits here on web ui, it's very ugly now with 5 commits for two changed files. I could just create a new clean pull request in case ;).

**Comment by Fourdee** [comment]
@MichaIng 

Perfect, great work, many thanks 👍 

> it's very ugly now with 5 commits for two changed files. I could just create a new clean pull request in case ;).

No need, its all good, we know what the extra commits are for 😄 I'll get this merged.

**Pull request merged**